src/isDeepEqual.test.ts

```diff
@@ -155,6 +155,64 @@ describe("arrays", () => {
   });
 });
 
+describe("Maps", () => {
```

Comment:
There's no test case that tests that two simple maps that are equal come up equal, e.g.
```ts
describe("Maps", () => {
  it('works on shallow equal maps', () => {
    expect(isDeepEqual(new Map([['a', 1]]), new Map([['a', 1]])).toBe(true);
  });
```